x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/cards/antivirus_registration_card.tsx

```diff
@@ -44,19 +50,73 @@ export const AntivirusRegistrationCard = memo<AntivirusRegistrationCardProps>(
   ({ policy, onChange, mode, 'data-test-subj': dataTestSubj }) => {
     const getTestId = useTestIdGenerator(dataTestSubj);
     const isProtectionsAllowed = !useGetProtectionsUnavailableComponent();
-    const isChecked = policy.windows.antivirus_registration.enabled;
     const isEditMode = mode === 'edit';
 
-    const handleSwitchChange = useCallback(
-      (event) => {
-        const updatedPolicy = cloneDeep(policy);
-        updatedPolicy.windows.antivirus_registration.enabled = event.target.checked;
+    let currentMode: AntivirusRegistrationModes;
+    if (policy.windows.antivirus_registration.mode) {
+      currentMode = policy.windows.antivirus_registration.mode;
+    } else {
+      currentMode = policy.windows.antivirus_registration.enabled
+        ? AntivirusRegistrationModes.enabled
+        : AntivirusRegistrationModes.disabled;
+    }
 
-        onChange({ isValid: true, updatedPolicy });
-      },
-      [onChange, policy]
+    const labels: Record<AntivirusRegistrationModes, React.ReactNode> = useMemo(
+      () => ({
+        [AntivirusRegistrationModes.disabled]: i18n.translate(
+          'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.disabled',
+          { defaultMessage: 'Disabled' }
+        ),
+        [AntivirusRegistrationModes.enabled]: i18n.translate(
+          'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.enabled',
+          { defaultMessage: 'Enabled' }
+        ),
+        [AntivirusRegistrationModes.sync]: (
+          <>
+            <EuiText size="s">
+              {i18n.translate(
+                'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.syncWithMalwarePrevent',
+                { defaultMessage: 'Sync with Malware protection level' }
+              )}{' '}
+              <EuiIconTip
+                position="right"
+                content={i18n.translate(
+                  'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.syncWithMalwarePrevent.tooltip',
+                  {
+                    defaultMessage:
+                      'Using this setting will automatically enable antivirus registration if Malware protection is set to prevent. ' +
+                      'In any other case antivirus registration will be disabled.',
+                  }
+                )}
+              />
+            </EuiText>
+            <EuiText color={isEditMode ? 'subdued' : undefined} size="xs">
+              {i18n.translate(
+                'xpack.securitySolution.endpoint.policy.details.antivirusRegistration.syncWithMalwarePrevent.currentOutcome',
+                {
+                  defaultMessage: '(with current Malware settings: {currentOutcome})',
```

Comment:
If this string is always expected to appear below the "Sync with malware protection level" string, I don't think we need to repeat "malware," just keep it simpler:
```suggestion
                  defaultMessage: '(Current level: {currentOutcome})',
```